nxc/protocols/winrm/proto_args.py

```diff
@@ -4,17 +4,26 @@
 def proto_args(parser, std_parser, module_parser):
     winrm_parser = parser.add_parser("winrm", help="own stuff using WINRM", parents=[std_parser, module_parser])
     winrm_parser.add_argument("-H", "--hash", metavar="HASH", dest="hash", nargs="+", default=[], help="NTLM hash(es) or file(s) containing NTLM hashes")
-    winrm_parser.add_argument("--port", type=int, default=0, help="Custom WinRM port")
-    winrm_parser.add_argument("--ssl", action="store_true", help="Connect to SSL Enabled WINRM")
-    winrm_parser.add_argument("--ignore-ssl-cert", action="store_true", help="Ignore Certificate Verification")
+    winrm_parser.add_argument("--port", nargs="+", default=["5985", "5986"], help="Custom WinRM port, default is %(default)s, format: 'http-port,https-port' or 'single-port'"
+                              "(http & https will use same port when given single port)")
+    winrm_parser.add_argument("--check-proto", choices={"both", "http", "https"}, default="both", help="Choose what prorocol you want to check, default is %(default)s")
```

Comment:
instead of "both", can you just make it a list to pass in, like `http https`, the same as the `--port` option?

Reply:
Sure!

Reply:
Fixed!